app/schema.py

```diff
@@ -44,6 +50,8 @@ class MentorUpdate(BaseModel):
     referred_by: Optional[constr(max_length=255)]
     other_info: Optional[constr(max_length=2500)]
     validate_status: Optional[constr(max_length=255)]
```

Comment:
validate_status in MentorUpdate collection needs to be changed from a string to stringLiteral to reflect same changes in Mentor collection